src/LLVM_Runtime_Linker.cpp

```diff
@@ -745,7 +743,8 @@ std::unique_ptr<llvm::Module> get_initial_module_for_target(Target t, llvm::LLVM
             modules.push_back(get_initmod_old_buffer_t(c, bits_64, debug));
             modules.push_back(get_initmod_errors(c, bits_64, debug));
 
-            if (t.arch != Target::MIPS && t.os != Target::NoOS) {
+            if (t.arch != Target::MIPS && t.os != Target::NoOS &&
+                t.os != Target::QuRT) {
```

Comment:
Hmm, does the profiler still work? I use it often on Hexagon, and it does work, it's very useful.

Reply:
I guess it probably does, since the profiler support code is still defined in the remote runtime?